04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -35,36 +38,44 @@ This lesson will walk through how to work with multi-band raster data in R.
 After completing this activity, you will:
 
 * Know how to identify a single vs. a multi-band raster file.
-* Be able to import multi-band rasters into `R` using the `raster` library.
-* Be able to plot multi-band color image rasters in `R` using `plotRGB`.
-* Understand what a `nodata` value is in rasters.
+* Be able to import multi-band rasters into `R` using the `raster` package.
+* Be able to plot multi-band color image rasters in `R` using `plotRGB()`.
+* Understand what a NoData value is in a raster.
```

Comment:
Yup - it's a point of confusion caused by different platforms spelling and formatting things differently!
Why don't we use `NoData` value everywhere unless we're explicitly referring to the tif tag? Does that sound ok? I don't think there's a right or wrong answer but we need consistency.